src/mainwindow.cpp

```diff
@@ -340,6 +317,10 @@ void MainWindow::onCancel()
 void MainWindow::onCut()
 {
     qDebug() << "Cut.";
+    // TODO it's not the best way to manage event.
+    // Shortcut are good to be kept customisable. If we do so in the future, we'll need to update all those methods.
+    // Future refacto : expose the event to call and call it directly.
+    // And from the other way, use QShortcut (we might need to set shortcutContext) instead of keypressevent
```

Comment:
It's not the best way to manage event.
Shortcut are good to be kept customisable. If we do so in the future, we'll need to update all those methods.
Future refacto : expose the event to call and call it directly.
And from the other way, use QShortcut (we might need to set shortcutContext) instead of keypressevent